package.json

```diff
@@ -10,6 +10,7 @@
     "test-fast": "mocha --slow 10 --timeout 15000 --reporter spec --recursive tests --bail --exclude \"tests/unit/relations/files/**\"",
     "test-opt": "mocha --slow 10 --timeout 15000 --reporter spec --recursive tests --bail --trace_opt --trace_deopt --exclude \"tests/unit/relations/files/**\"",
     "test-prof": "mocha --slow 10 --timeout 15000 --reporter spec --recursive tests --prof --exclude \"tests/unit/relations/files/**\"",
+    "test-sqlite": "cross-env DATABASES=sqlite3 mocha --slow 10 --timeout 15000 --reporter spec --recursive tests --exclude \"tests/unit/relations/files/**\"",
```

Comment:
This was added for the convenience of local debugging, more often than not sqlite is quite enough.